db/schemaGenerator/generate_schema.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
+const Promise = require('bluebird');
+Promise.promisifyAll(fs);
+const {
+    product_descriptions_Inserts,
+    benefits_Inserts,
+    details_Inserts } = require('./insert_query_generators');
+
+
+var schemaData = 
+`DROP DATABASE IF EXISTS fec;\n\n` +
+
+`CREATE DATABASE fec;\n\n` +
+
+`USE fec;\n\n` +
+
+`CREATE TABLE product_descriptions (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    description VARCHAR(250) NOT NULL,
+    title VARCHAR(100) NOT NULL
+);\n\n` +
+
+`CREATE TABLE benefits (
```

Comment:
Unless you really want to do it this way, I'd suggest drastically simplifying this with a single table with fields for description, benefits, and details (1:1 with the "style" table) which you could work on generating a larger text blocks. I'm having trouble imagining a situation where it would benefit anyone in the system to have every individual "benefit" or "detail" in its own table row.

Reply:
I was thinking of having multiple benefits for each product, and multiple details for each product. 
I attached a snapshot of a nike shoe with a couple of benefits and one detail. But some of them have more than one detail.

![benefits Details](https://user-images.githubusercontent.com/55673139/76323280-c0ac5e80-62a1-11ea-9564-c6bed9b96f04.png)

Reply:
Looks great, I don't have any feedback on this file.

Reply:
MY FINAL PITCH THEN I SHUT UP ABOUIT IT:

IMHO it's simpler to have your generator produce the whole section as one text block (like a human content editor would probably do) rather than dice each sentence into its own coordinated table row, then programmatically try to piece them back together at runtime. 

But obviously it's your module, and if you prefer it the multi--table way, either as a database challenge, or if you see some additional value in having your style description stored like that, I will not get in your way :)

Reply:
@damienmjones Ohhhhh I see what you mean. That's true, i can have all benefits and details as one string. And then parse that string on the client side. That's a good idea. Especially since text like this will rarely change.